kmip/pie/client.py

```diff
@@ -138,7 +138,8 @@ def close(self):
                 self.logger.exception("could not close client connection", e)
                 raise e
 
-    def create(self, algorithm, length, operation_policy_name=None, name=None):
+    def create(self, algorithm, length, operation_policy_name=None, name=None,
+               crypto_usage_masks=list()):
```

Comment:
I'm fine using the full name:
`crypto_usage_masks` -> `cryptographic_usage_mask`